src/app/dao/tests/UserDAO.spec.ts

```diff
@@ -66,4 +68,16 @@ describe('UserDAO', () => {
             await FireAuth.signOut(TestBed.inject(FireAuth.Auth));
         });
     });
+    describe('updatePresenceToken', () => {
+        it('should delegate to update', async() => {
+            // Given any situation
+            spyOn(dao, 'update');
+
+            // When calling updatePresenceToken
+            await dao.updatePresenceToken('joser');
```

Comment:
joser is an id! -> joserId

Reply:
aurais JOSER avoir la même ID ?

Reply:
sinon ça fait un peu roger-rabbit, joserId